.github/workflows/ci.yml

```
@@ -88,40 +92,46 @@ jobs:
         docker rmi $(docker image ls -aq)
         df -h
 
-    - name: Deploy chart
+    - name: Deploy chart ☕️
       run: |
-        helm dependency update ${CHART}
-        helm install \
-          --set rasax.disableTelemetry=true \
-          --set duckling.enabled=false \
-          --set rasax.initialUser.password=${INITIAL_USER_PASSWORD} \
-          --set nginx.service.type=ClusterIP \
-          --set ingress.hosts[0].host=,ingress.hosts[0].paths={/} \
-          ${RELEASE_NAME} ${CHART}
-
-    - name: Wait for deployment to be ready
+        helmfile -f .github/deployments/helmfile.yaml sync
+
+    - name: Set host name and address env variables 🖌
+      run: |
+        ADDRESS=$(kubectl get ingress  -o json | jq .items[0].status.loadBalancer.ingress[0].ip)
+        HOST=$(kubectl get ingress  -o json | jq .items[0].spec.rules[0].host)
```

```suggestion
        ADDRESS=$(kubectl get ingress -o jsonpath='{.items[0].status.loadBalancer.ingress[0].ip}')
        HOST=$(kubectl get ingress -o jsonpath='{.items[0].spec.rules[0].host}')
```

You don't have to use `jq`, `kubectl` can be used to get a value of a specific path